services/frontend-service/src/ui/Releases.tsx

```diff
@@ -230,6 +231,7 @@ const ApplicationBox: React.FC<any> = (props: {
                 {warnings}
                 {name}
                 {undeployButton}
+                {application.team !== '' ? 'Owner: ' + application.team : null}
```

Comment:
Can you give the team name a different style. Maybe a lighter gray or something?

Reply:
updated